static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,113 @@
+a storage server, downloading them with `wget` in our examples. This setup was
+then revised to download the dataset with `dvc get` instead, so we created the
+[dataset-registry](https://github.com/iterative/dataset-registry)) project, a
+<abbr>DVC project</abbr> hosted on GitHub, to version the dataset (see its
+[`tutorial/ver`](https://github.com/iterative/dataset-registry/tree/master/tutorial/ver)
+directory).
+
+However, there are a few problems with the way this dataset is structured (in 2
+parts). Most importantly, this single dataset is tracked by 2 different
+[DVC-files](/doc/user-guide/dvc-file-format), instead of 2 versions of the same
+one, which would better reflect the intentions of this dataset... Fortunately,
+we have also prepared an improved alternative in the
+[`use-cases/`](https://github.com/iterative/dataset-registry/tree/master/use-cases)
+directory of the same repository.
+
+As step one, we extracted the first part of the dataset into the
+`use-cases/cats-dogs` directory (illustrated below), and ran <code>dvc add
+use-cases/cats-dogs</code> to
+[track the entire directory](https://dvc.org/doc/command-reference/add#example-directory).
+
+```dvc
+$ tree use-cases/cats-dogs --filelimit 3
+use-cases/cats-dogs
+└── data
+    ├── train
+    │   ├── cats [500 image files]
+    │   └── dogs [500 image files]
+    └── validation
+        ├── cats [400 image files]
+        └── dogs [400 image files]
+```
+
+This first version uses the
+[`cats-dogs-v1`](https://github.com/iterative/dataset-registry/tree/cats-dogs-v1/use-cases)
+Git tag. In a local DVC project, we can obtain this dataset with the following
+command (note the usage of `--rev`):
+
+```dvc
+$ dvc import --rev cats-dogs-v1 \
+             git@github.com:iterative/dataset-registry.git \
+             use-cases/cats-dogs
+```
+
+> Note that unlike `dvc get`, which can be used from any directory, `dvc import`
+> always needs to run from an [initialized](/doc/command-reference/init) DVC
+> project.
+
+Importing keeps the connection between the local project and data registry where
+we are downloading the dataset from. This is achieved by creating a special
+DVC-file (a.k.a. an _import stage_) – which can be used for versioning the
+import with Git in the local project. This connection will come in handy when
+the source data changes, and we want to obtain these updates...
+
+Back in our **dataset-registry** repository, the second (and last) version of
+our dataset exists under the
+[`cats-dogs-v2`](https://github.com/iterative/dataset-registry/tree/cats-dogs-v2/use-cases)
+tag. It was created by extracting the second part of the dataset, with 1000
+additional images (500 cats, 500 dogs) in the same directory structure, and
+simply running <code>dvc add use-cases/cats-dogs</code> again.
+
+In our local project, all we have to do in order to obtain this latest version
+of the dataset is to run:
+
+```dvc
+$ dvc update cats-dogs.dvc
```

Comment:
Pending review of this command in the example once iterative/dvc/issues/2650 is resolved.

Reply:
Addressed.